cli.js

```diff
@@ -31,21 +31,21 @@ if (args.version) {
 }
 
 if (args.help || args._.length === 0) {
```

Comment:
Can we replace `minimist` with https://www.npmjs.com/package/arg? I've generally found it to be more robust and type-safe (no implicit arrays).

Reply:
I have no real opinion on this